src/main/java/xyz/nucleoid/plasmid/game/stats/StatisticKey.java

```diff
@@ -0,0 +1,84 @@
+package xyz.nucleoid.plasmid.game.stats;
+
+import com.google.gson.JsonObject;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.StringIdentifiable;
+import xyz.nucleoid.plasmid.game.stats.key.DoubleStatisticKey;
+import xyz.nucleoid.plasmid.game.stats.key.FloatStatisticKey;
+import xyz.nucleoid.plasmid.game.stats.key.IntStatisticKey;
+
+/**
+ * Type-safe keys for identifying statistics.
+ * <p>
+ * Ideally should have a corresponding translation key in the form <code>statistic.[bundle namespace].[key namespace].[key path]</code>
```

Comment:
Should statistic translation keys involve the bundle namespace? I imagine the same statistic key would always mean the same thing irrespective of what game it was involved in?

Reply:
On the line below it says that the translation key can also be `statistic.[key namespace].[key path]`, which could be used as a fallback if there is not a `statistic.[bundle namespace].[key namespace].[key path]` translation.
This would allow a game like bedwars to potentially store two bundles, such as `bedwars` and `bedwars-teams`, both containing a value for the statistic `plasmid:wins`, but can override the display name for the `bedwars-teams` bundle value to be `Wins (teams)` without requiring another statistic key